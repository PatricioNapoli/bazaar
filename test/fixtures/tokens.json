[
  {"address":"0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2","symbol":"WETH","name":"Wrapped Ether","decimals":"18"},
  {"address":"0x476c5e26a75bd202a9683ffd34359c0cc15be0ff","symbol":"SRM","name":"Serum","decimals":"6"}
]
