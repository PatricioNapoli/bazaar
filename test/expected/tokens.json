{"0x476c5e26a75bd202a9683ffd34359c0cc15be0ff":{"Address":"0x476c5e26a75bd202a9683ffd34359c0cc15be0ff","Symbol":"SRM","Name":"Serum","Decimals":"6"},"0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2":{"Address":"0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2","Symbol":"WETH","Name":"Wrapped Ether","Decimals":"18"}}